my-react-app/src/App.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import Profilecard from './Profilecard/Profilecard';
+import Tabs from './Tabs/Tabs';
 function App() {
   return (
     <div>
-<Profilecard/>
+
+<Tabs/>
```

Comment:
Always use proper formatting. We can install prettier tool in our vs code to format our code on save